testhelpers/featureContext.go

```diff
@@ -10,9 +10,13 @@ func FeatureContext(s *godog.Suite) {
 	s.BeforeScenario(ctx.SetupTestContext)
 
 	s.Step(`^the database has the following table \'([\w\-_]*)\':$`, ctx.DBHasTable)
+	s.Step(`^the table "([^"]*)" has a unique key "([^"]*)" on "([^"]*)"$`, ctx.TableHasUniqueKey)
```

Comment:
I wouldn't have polluted the current code to simulate what the future DB should be. I think this is typically the kind of stuff that will be difficult to understand in a few months.

Reply:
We can remove this line as soon as the DB is upgraded. I think it would be better to keep the scenario although the DB is not ready yet, because this scenario covers the feature well.
Or do you want me to remove it anyway?

Reply:
I would not have put it, but ok you can leave it if you prefer.